example/brunel/brunel_miniapp.cpp

```diff
@@ -131,12 +131,7 @@ class brunel_recipe: public recipe {
         time_type t0 = 0;
         cell_member_type target{gid, 0};
 
-        gens.push_back(arb::make_event_generator<pgen>(
-                        target,
-                        weight_ext_,
-                        G,
-                        t0,
-                        lambda_));
+        gens.emplace_back(pgen(target, weight_ext_, G, t0, lambda_));
```

Comment:
This is not so important, but out of curiosity: would constructing target (a pair) in-place with {gid, 0} avoid copying it and make it a tiny more efficient?

Reply:
`cell_member_type` is a POD type, so I don't expect it would make any difference.
http://en.cppreference.com/w/cpp/concept/PODType